docs/benchmarks.md

```diff
@@ -1,131 +1,183 @@
+### Results
+
+#### Sequential Reads
+
+| File Size | BlockSize | Bandwidth in (MiB/sec) | Avg Latency (msec) | IOPs     |
+|-----------|-----------|------------------------|--------------------|----------|
+| 128KB     | 128K      | 862                    | 18.54              | 6898.27  |
+| 256KB     | 128K      | 1548                   | 10.325             | 12386.03 |
+| 1MB       | 1M        | 5108                   | 24.99              | 5113.21  |
+| 5MB       | 1M        | 7282                   | 17.505             | 7308.51  |
+| 10MB      | 1M        | 7946                   | 16.092             | 7946.63  |
+| 50MB      | 1M        | 7810                   | 16.356             | 7818.17  |
+| 100MB     | 1M        | 7839                   | 16.295             | 7840.17  |
+| 200MB     | 1M        | 7879                   | 16.217             | 7884.45  |
+| 1GB       | 1M        | 7911                   | 16.162             | 7910.19  |
+
+#### Random Reads
+
+| File Size | BlockSize | Bandwidth in MiB/sec | Avg Latency (msec) | IOPs     |
+|-----------|-----------|----------------------|--------------------|----------|
+| 256KB     | 128K      | 1264                 | 12.648             | 10109.62 |
+| 5MB       | 1M        | 4367                 | 29.129             | 4449.03  |
+| 10MB      | 1M        | 3810                 | 33.496             | 3825.54  |
+| 50MB      | 1M        | 4370                 | 29.185             | 4426.73  |
+| 100MB     | 1M        | 3504                 | 36.421             | 3505.01  |
+| 200MB     | 1M        | 3048                 | 41.919             | 3044.43  |
+| 1GB       | 1M        | 2120                 | 60.246             | 2114.33  |
+
+## Writes
+
+### FIO spec
+
+  ```
+    [global]
+    ioengine=sync
+    direct=1
+    fadvise_hint=0
+    verify=0
+    iodepth=64
+    invalidate=1
+    time_based=0
+    file_append=0
+    # By default fio creates all files first and then starts writing to them. This option is to disable that behavior. 
+    create_on_open=1 
+    thread=1
+    openfiles=1
+    group_reporting=1
+    allrandrepeat=1
+    # Every file is written only once. Set nrfiles per thread in such a way that the test runs for 1-2 min. 
+    # This will vary based on file size. Change the value from table to get provided results.
+    nrfiles=2
+    filename_format=$jobname.$jobnum.$filenum
+    # Change this to randwrite to test random writes.
+    rw=write   
+    # Update the block size value from the table for different 
+    bs=1M
+    # Update the file size value from table(file size) for different experiments.
+    filesize=1G  
+    [experiment]
+    stonewall
+    # Change the test directory (1mb) for different experiments. The directory must exist within the mounted directory.
+    directory=gcs/1gb
+    numjobs=112
+ ```
+
+**Note:** Performance will be better if you experiment on scenarios where files
```

Comment:
"Benchmarking is done writing out new files. Performance will be different for writes to existing files, as this first requires a full file download first"